src/utils/forRender.js

```diff
@@ -0,0 +1,26 @@
+import Iso from 'iso'
+import * as Render from './Render'
+
+// XXX include the promise polyfill?
+
+export const withData = Render.withData
+
+export default function forRender(alt, Component, props) {
+  // recycle state
+  alt.recycle()
+
+  /*eslint-disable */
+  if (typeof window === 'undefined') {
+    /*eslint-enable */
```

Comment:
yeah. I'm going to either remove browser: true or remove these.

I just threw browser: true in there so lint would shut up so I could push this.